### ISSUE TITLE
[CI] Torch CI Camp

### ISSUE THREAD

**Issue by Groverkss** [iree-org/iree#22584]
We recently ramped up two new CIs based on need: torch_models and torch_ops: https://github.com/iree-org/iree-test-suites/tree/main/torch_models and https://github.com/iree-org/iree-test-suites/tree/main/torch_ops . This issue tracks pending tasks to finish this migration.

The way this issue is structured is each task not taken by anyone has a (Volunteer needed) tag in front of it. It is expected that people will volunteer for issues and we will collectively own the state of the CI.

## SharktankModels CI migration to Torch Models CI

We were previously running model tests in the shark_models ci. This CI had it's issues which are mentioned in the torch models ci init pr: https://github.com/iree-org/iree-test-suites/pull/113 . 

Example migration PR: https://github.com/iree-org/iree/pull/22364

Some instructions on how to add a test are here: https://github.com/iree-org/iree/tree/main/tests/external/iree-test-suites/torch_models

- [x] Move SDXL Punet int8 fp16 attn @MaheshRavishankar  https://github.com/iree-org/iree/pull/22364
- [ ] Move SDXL Punet int8 fp8 attn (Volunteer needed)
- [ ] Move SDXL Unet (Volunteer needed)
- [ ] Move SDXL Vae (Volunteer needed)
- [ ] Move SDXL Clip (Volunteer needed)
- [ ] Move SD3 MMDIT (Volunteer needed)
- [ ] Move SDXL e2e (Volunteer needed)
- [x] Add llama 8b fp16 quality test @Groverkss https://github.com/iree-org/iree/pull/22379
- [ ] Add llama 8b fp8 quality test (Volunteer needed)
- [ ] Add data tiling llama 8b fp16 (Volunteer needed)
- [ ] Add data tiling llama 8b fp8 (Volunteer needed)

## Torch Ops CI

TODO: Add more instructions once this CI is enabled

- [ ] Add quality tests for GEMMs (Volunteer needed)
- [ ] Add quality tests for Convs (Volunteer needed)
- [ ] Add quality tests for Layer Norm (Volunter needed)
- [ ] Add quality tests for Batch Norm (Volunteer needed)
- [ ] Add quality tests for Softmax (Volunteer needed)
- [ ] Add quality tests for Attention (Volunteer needed)
- [ ] Add quality tests for PagedAttention (Volunteer needed)
- [ ] Add quality tests for TopK (Volunteer needed)
- [ ] Add quality tests for Argmax (Volunteer needed)

## Torch Models CI Meta Tasks

These are more meta tasks relating to the actual ci

- [ ] Promote torch_models ci to presubmit after migration
- [ ] Remove sharktank_models ci for models after migration
- [ ] Add torch_models ci for MI250 (Volunteer needed)
- [ ] Add torch_models ci for MI308 (Volunteer needed)
- [ ] Add torch_models ci for MI355 (Runner has issues)
- [ ] Add torch_models ci for W7900 (Volunteer needed)
- [ ] Add torch_models ci for XTX7900 (Volunteer needed)
- [ ] Add torch_models ci for RDNA4 (Need runner)
